### PR TITLE
Lib merge additional options 

### PR DESCRIPTION
**Pull request by jidur**

### CODE REVIEW

File: lib/npg_seq_melt/sample_merge.pm [hunk 1]
```diff
@@ -35,12 +37,14 @@ with qw{
...
-Readonly::Scalar my $SAMTOOLS      => q[samtools1];
-Readonly::Scalar my $SUMMARY_LINK   => q{Latest_Summary};
-Readonly::Scalar my $MD5SUB => 4;
+Readonly::Scalar my $P4_COMMON_TEMPLATE  => q[alignment_common.json];
+Readonly::Scalar my $VIV_SCRIPT          => q[viv.pl];
+Readonly::Scalar my $VTFP_SCRIPT         => q[vtfp.pl];
+Readonly::Scalar my $SAMTOOLS            => q[samtools1];
+Readonly::Scalar my $SUMMARY_LINK        => q{Latest_Summary};
+Readonly::Scalar my $BATON               => q{~dj3/repos/baton/install/bin/baton};
+Readonly::Scalar my $BATON_LIST          => q{~dj3/repos/baton/install/bin/baton-list};
```
Comment (mgcam) [inline lib/npg_seq_melt/sample_merge.pm]: Is it possible to change to installed baton and have just 'baton' and 'baton-list'

File: lib/npg_seq_melt/sample_merge.pm [hunk 2]
```diff
@@ -676,6 +718,33 @@ sub _build__source_cram {
...
 
+=head2 get_irods_hostname
+
+=cut
+
+sub get_irods_hostname{
+    my $self       = shift;
+    my $irods_collection = shift; #/seq/id_run/rpt.cram
+    my $index            = shift; #0 or 1
+    my $cmd = qq[$BATON -c $irods_collection | $BATON_LIST --replicate];
```
Comment (mgcam) [inline lib/npg_seq_melt/sample_merge.pm]: Is it possible/sensible to use baton perl wrapper?

File: lib/npg_seq_melt/sample_merge.pm (hunk 1 above)
Comment (jidur) [inline lib/npg_seq_melt/sample_merge.pm]: It is working now with the latest /software/npg/20151019, so I will remove the dj3 paths

File: lib/npg_seq_melt/sample_merge.pm (hunk 2 above)
Comment (jidur) [inline lib/npg_seq_melt/sample_merge.pm]: We had been thinking of it, when available

Comment (jidur) [inline lib/npg_seq_melt/sample_merge.pm]: Keith confirmed it is now available in the current installation...

Comment (mgcam) [inline lib/npg_seq_melt/sample_merge.pm]: On 21/10/2015 16:45, jidur wrote:
> Keith confirmed it is now available in the current installation...
Yes. However, let's change to his perl wrapper in a separate pull request.



-- 
 The Wellcome Trust Sanger Institute is operated by Genome Research 
 Limited, a charity registered in England with number 1021457 and a 
 company registered in England with number 2742969, whose registered 
 office is 215 Euston Road, London, NW1 2BE.

File: lib/npg_seq_melt/sample_merge.pm (hunk 1 above)
Comment (mgcam) [inline lib/npg_seq_melt/sample_merge.pm]: On 21/10/2015 16:21, jidur wrote:
> It is working now with the latest /software/npg/20151019, so I will 
> remove the dj3 paths
Excellent!


-- 
 The Wellcome Trust Sanger Institute is operated by Genome Research 
 Limited, a charity registered in England with number 1021457 and a 
 company registered in England with number 2742969, whose registered 
 office is 215 Euston Road, London, NW1 2BE.

**Pull request merged**